### PR TITLE
Reworded phrases and fixed grammar

### PR DESCRIPTION
**Pull request by Conflate**
-Reworded phrases and fixed awkward wording
-fixed grammatical errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-You'll also need the wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon and possibly others. 
+To build and run this project, you will require [build ea+44 of project panama on jdk 13](https://jdk.java.net/panama/).
+The following will be required as well: the wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon and possibly others. 
 
 ## Building
 
-To build this project, you can run `sbt compile` in the root of the project directory. If the build fails, please check to make sure that these settings in build.sbt
-
+To build this project, run `sbt compile` in the root of the project directory. 
+If the build fails, please check to make sure that the following settings are in build.sbt
```
Comment (markehammons) [inline README.md]: The settings will assuredly be in build.sbt, the issue is that they may not be set to the right paths. Please change the wording from "the following settings are in build.sbt" to "the following settings in build.sbt" to allow this line to flow into the bottom "point to your xdg-shell-unstable-v6.xml..."  line.

File: README.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ libraryDirectory := file("/usr/lib64")
 point to your xdg-shell-unstable-v6.xml, your include path, and the path that contains the .so files on your system.
 
 ### Note for Part 3
-I had to add the file xdg-shell-unstable-v6-protocol.h from a manual build of wlroots to get this step able to work.
-My distribution's wlroots-devel package does not contain this file, despite the wlr_xdg_shell_v6.h header and depending
-on this protocol
+The file, xdg-shell-unstable-v6-protocol.h, from a manual build of wlroots was necessary for this step to function correctly.
```
Comment (markehammons) [inline README.md]: This isn't actually necessary anymore, the build system generates the xdg-shell-unstable-v6-protocol.h file automatically on compile. Please go ahead and remove this note entirely as it's no longer relevant.

**Comment by markehammons** [comment]
I've noted some changes I'd like made to the updated readme. If you could implement those, I'd appreciate it. Thanks!

**Comment by Conflate** [comment]
Fixed as you asked unless I misunderstood something

**Comment by Conflate** [comment]
I'd really appreciate it if you could merge before March 22nd 11:59 PM PST since I need the successful pull request for an assignment.

**Pull request merged**